src/main/java/teammates/ui/webapi/action/GetAuthInfoAction.java

```diff
@@ -40,7 +41,9 @@ public ActionResult execute() {
                     gateKeeper.getLoginUrl(frontendUrl + Const.WebPageURIs.ADMIN_HOME_PAGE)
             );
         } else {
-            output = new AuthInfo(userInfo, authType == AuthType.MASQUERADE,
+            String googleId = userInfo.getId();
+            AccountAttributes accountInfo = logic.getAccount(googleId);
+            output = new AuthInfo(userInfo, accountInfo.getInstitute(), authType == AuthType.MASQUERADE,
```

Comment:
If `accountInfo` is null, you're going to hit NPE here. Need to add a guard.

Reply:
Fixed!